src/main/dumpxdr.cpp

```diff
@@ -116,6 +118,27 @@ xdr_printer(const PublicKey& pk)
     return KeyUtils::toStrKey<PublicKey>(pk);
 }
 
+std::string
+xdr_printer(const MuxedAccount& muxedAccount)
```

Comment:
Nitpick: same thing about consistent-const here and line 116.

Reply:
Done!